src/sentry/static/sentry/app/components/charts/releaseSeries.jsx

```diff
@@ -73,14 +44,55 @@ class ReleaseSeries extends React.Component {
       .then(releases => {
         this.setState({
           releases,
-          releaseSeries: [getReleaseSeries(releases)],
+          releaseSeries: [this.getReleaseSeries(releases)],
         });
       })
       .catch(() => {
         addErrorMessage(t('Error fetching releases'));
       });
   }
 
+  getReleaseSeries = releases => {
+    const {organization, router} = this.props;
+
+    return {
+      seriesName: 'Releases',
+      data: [],
+      markLine: MarkLine({
+        lineStyle: {
+          normal: {
+            color: theme.purpleLight,
+            opacity: 0.3,
+            type: 'solid',
+          },
+        },
+        tooltip: {
+          formatter: ({data}) => {
+            return `<div>${getFormattedDate(data.value, 'MMM D, YYYY LT')} <br />
+            Release: ${data.name}<br />
+            </div>`;
+          },
+        },
+        label: {
+          show: false,
+        },
+        data: releases.map(release => ({
+          xAxis: +new Date(release.dateCreated),
+          name: release.shortVersion,
+          value: release.shortVersion,
+          onClick: () => {
```

Comment:
This was the only thing added

Reply:
Old route meaning whatever route `ReleaseSeries` is on? Yes, since we could be on Events before clicking into release details from the chart.

Reply:
I think Lyn meant the project release page. Is there a scenario where a user will have event stream but not the sentry10 flag?

Reply:
🤦‍♂️ Sorry - there shouldn't be a reason to link to the project specific releases. AFAIK, everyone will be able to access org releases (and only certain users can utilize multi project).

Reply:
In future yes, but as of today don't users have events access but not org releases? So it will be broken immediately when we merge?

Reply:
fixed https://github.com/getsentry/sentry/pull/11654/commits/39788d549a0dabb29d10e54411501214632c6faf